package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "smash_tv",
+  "version": "1.0.0",
+  "description": "Wow Dashboard for TV",
+  "main": "index.html",
+  "repository": "git@github.com:customink/smash-tv.git",
+  "author": "Chris Mar <cmar@customink.com>",
+  "license": "UNLICENSED",
+  "private": true,
+  "scripts": {
+    "clean": "rm -rf dist",
+    "build": "parcel build --public-url ./",
+    "server": "parcel",
+    "deploy": "yarn run clean && yarn run build && mv dist/smash_tv.html dist/index.html && gh-pages -d dist"
```

Comment:
kind of anoying quirk of parcel.js is that it uses the package.name for the output file without an option to use something else so I'm manually renaming it to index.html for deploying to github